Makefile

```diff
@@ -73,9 +73,9 @@ FUZZER_BUILD ?= 0
 # NOTE: when changing any value below, you'll need to adjust WORKSPACE file by hand:
 #         - uncomment source build section, comment binary section
 #         - adjust binary version path - version variable is not passed to WORKSPACE file!
-OV_SOURCE_BRANCH ?= c328ded614fad2b03d598754c3735f44ae2ecb82  # master 2024-08-20
-OV_CONTRIB_BRANCH ?= e6eb43a32c98a04162a921a80d89f82b30910973  # master 2024-06-13
-OV_TOKENIZERS_BRANCH ?= 9a4d2a9c447959e6bd26384f34cfd83b3275fe64  # master 2024-08-20
+OV_SOURCE_BRANCH ?= 98188ad2efa74d3b73ec6d2e5bd6ac80c4fdb570  # releases/2024/4 2024-08-29
+OV_CONTRIB_BRANCH ?= e6eb43a32c98a04162a921a80d89f82b30910973  # releases/2024/4 2024-06-13
```

Comment:
is it really release branch? Sha is unchanged since june

Reply:
yes, it is